enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -247,7 +247,8 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
     } catch (ComponentLookupException e) {
       throw new EnforcerRuleException("Unable to lookup a component " + e.getLocalizedMessage(), e);
     } catch (DependencyResolutionException e) {
-      throw new EnforcerRuleException("Unable to build a dependency graph", e);
+      throw new EnforcerRuleException(
+          "Unable to build a dependency graph: " + e.getLocalizedMessage(), e);
```

Comment:
probably just e.getMessage. I don't think I've ever seen anyone localize one of these, and we haven't localized our own rules.

Reply:
Updated to getMessage